### PR TITLE
improve sample.config.php parameter descriptions

### PR DESCRIPTION
**Pull request by christianlupus**
This is a replacement for nextcloud/documentation#9166. It must be solved here in the server repo.

* Resolves: # 

## Summary
This just rearranges the sample config to obtain a better structure in terms of _sections_ in the documentation.

## TODO

- [ ] [One change](https://github.com/nextcloud/documentation/pull/9166/files#diff-f5b9c9ee7b52f546f39c5bb962cf7bc10bcacbad7972d5cc7a9373de9b9c7cf2L2539-R2542) was not yet incorporated

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included -> no changes made, so no tests written
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by kesselb** [comment]
Hi, can you use a self-explaining commit message to summarize the changes?

https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

**Comment by christianlupus** [comment]
I can offer to amend the commit with a new commit message. You mean to summarize the changes in the commit message, that it your point here, right?

**Comment by kesselb** [comment]
> I can offer to amend the commit with a new commit message. 

Amend or rebase as you like.

> You mean to summarize the changes in the commit message, that it your point here, right?

Yes. "Copy modifications from ..." is not understandable without following the link. Please summarize what you did (e.g. "improve readability of config sample file ....") .

**Review by kesselb — APPROVED** [review]

File: config/config.sample.php [hunk 1]
```diff
@@ -2291,9 +2288,13 @@
  *
  * For instance, if the phone property should default to the private scope
  * instead of the local one:
- * [
- *   \OCP\Accounts\IAccountManager::PROPERTY_PHONE => \OCP\Accounts\IAccountManager::SCOPE_PRIVATE
- * ]
+ * 
+ * ::
```
Comment (szaimen) [inline config/config.sample.php:2292]: is this change correct?

**Review by szaimen — APPROVED** [review]
LGTM, just one small question

File: config/config.sample.php (hunk 1 above)
Comment (christianlupus) [reply config/config.sample.php:2292]: This should make that part a verbatim text. I can check what happens after doing the static parsing and code generation in sphinx. Let me have a look.

Comment (christianlupus) [reply config/config.sample.php:2292]: I just built the documentation locally and looked at the output:
image

I do not see any issues here. What exactly were you referring to, @szaimen? The `::`? Or the indentation? Or something even different that I changed in fact without intention?

**Review by szaimen — APPROVED** [review]

**Pull request merged**